src/lib/program.ts

```diff
@@ -171,6 +171,10 @@ iam
   .option('-a, --account <alksAccount>', 'alks account to use')
   .option('-r, --role <alksRole>', 'alks role to use')
   .option('-F, --favorites', 'filters favorite accounts')
+  .option(
+    '--tags <tags...>',
```

Comment:
Both 't' and 'T' are taken for short flag values. Any suggestions on an alternative?